### ISSUE TITLE
Incorrect Qeq charges

### ISSUE THREAD

**Issue by danieleongari** [openbabel/openbabel#1899]
- [x] I believe this to be a bug with Open Babel
- [ ] This is a feature request

## Environment Information
Open Babel version: 2.3.2
Operating system and version: Ubuntu 18.04.1

## Expected Behavior

Qeq charges computed with openbabel for IRMOF-1 (also named MOF-5) coherent with the values computed with the same protocol as implemented in the code provided by B. Wells et al. (https://pubs.acs.org/doi/suppl/10.1021/jp510415h, see archive in the Supporting Information)

## Actual Behavior

Qeq charges are completely different, also unphysical: e.g., negative on the Zn ion (-2.66 while the correct one should be +0.46).

## Steps to Reproduce

obabel IRMOF-1.cif --partialcharges -O IRMOF-1_qeq-obabel.mol2
Output and comparison: [IRMOF-1.zip](https://github.com/openbabel/openbabel/files/2479296/IRMOF-1.zip)

**Comment by HAlireza94** [comment]
Hi, could you solve this problem?

**Comment by danieleongari** [comment]
@HAlireza94 No, I ended up using this code I ported on Github: https://github.com/danieleongari/egulp
Also, you may want to read our comparison of different Qeq variants: https://doi.org/10.1021/acs.jctc.8b00669

**Comment by HAlireza94** [comment]
I am working on MoS2 and trying to get qeq from openbabel but it gives me very uncorrect charges.
Hope this github would work for me.
Could you please give me any hint on using that c++ code provided in that github link?

**Comment by HAlireza94** [comment]
I wanted to install it but it says:
egulp.c:1:10: fatal error: 'gsl/gsl_linalg.h' file not found
#include <gsl/gsl_linalg.h>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [egulp.o] Error 1

**Comment by ltalirz** [comment]
Hey @HAlireza94 , looks like you are missing the gsl that is mentioned in the installation instructions (or your compiler does not find it)

**Comment by ltalirz** [comment]
Have a look at the [Makefile](https://github.com/danieleongari/egulp/blob/master/src/Makefile) to better understand how the compiler is trying to compile the program

**Comment by HAlireza94** [comment]
Hi thank you so much.
I fixed it a couple of minutes ago.
I just rewrite Makefile based on my gsl directory and liked them all.

**Comment by HAlireza94** [comment]
One more question, what is GMP param? what are the names of the columns?
I am sorry to ask a lot of questions.

**Comment by danieleongari** [comment]
You are now asking questions about the methodology, not the program: there is no quick answer.
I think the reference paper I posted before provides a good introductions to the QEq methods and on the choice of the parameters.

**Comment by HAlireza94** [comment]
Thank you so much!

**Comment by HAlireza94** [comment]
> You are now asking questions about the methodology, not the program: there is no quick answer. I think the reference paper I posted before provides a good introductions to the QEq methods and on the choice of the parameters.

Hi,
I just wanted to thank you so much.
I read your article and figured out what was going on.
now, i can get reasonable partial charges.
